sales_team_invoiced_target_domain/model/crm_team.py

```diff
@@ -0,0 +1,30 @@
+import ast
+
+from odoo import fields, models
+from odoo.osv import expression
+
+
+class CrmTeam(models.Model):
+    _inherit = "crm.team"
+
+    def _compute_invoiced(self):
+        if not self:
+            return
+        sales_team_invoiced_domain = self.env.company.sales_team_invoiced_domain
+        if not sales_team_invoiced_domain:
+            return super()._compute_invoiced()
+        today = fields.Date.today()
```

Comment:
Please use context_today to prevent errors in countries where the timezone is negative.

```suggestion
        today = fields.Date.context_today(self)
```